src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1027,7 +1027,7 @@ private Symbol UnwrapAlias(Symbol symbol, out AliasSymbol alias, DiagnosticBag d
                     diagnostics);
             }
 
-            if (options.IsAttributeTypeLookup())
+            if (Compilation.LanguageVersion < LanguageVersion.CSharp8 && options.IsAttributeTypeLookup())
             {
                 // Generic type cannot be an attribute type.
```

Comment:
>cannot [](start = 32, length = 6)

`// Generic type previously could not be an attribute type.`

Reply:
Done